catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/Subscribe.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.spatial.ogc.csw.catalog.common;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import net.opengis.cat.csw.v_2_0_2.GetRecordsType;
+
+/**
+ * JAX-RS Interface to define an OGC Catalogue Service for Web (CSW).
+ */
+@Path("/subscription")
+public interface Subscribe {
+
+    /**
+     * Deletes an active subscription
+     *
+     * @param requestId the requestId of the subscription to be removed
+     * @return Acknowledgment   returns a CSW Acknowledgment message with the subscription that was
+     * deleted or an empty 404 if none are found
+     * @throws CswException
```

Comment:
Yeah this particular CswException can be removed since nothing actually throws it anymore, but Ill look into the others it appears to default to a 400 status so their may be a couple places that I need to change it to 500 for server errors